### PR TITLE
[14.0] [MIG] hr_grade_rank: Migrated the module in v14.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**
- Module developed by **Pragati Patel**. (Assigned by **Yogesh Mahera**)

### CODE REVIEW

**Comment by Chandresh-SerpentCS** [comment]
@YogeshMahera-SerpentCS Please review the PR. Thanks.

**Comment by JayVora-SerpentCS** [comment]
@YogeshMahera-SerpentCS do spend 10 mins for 1 PR daily.

File: hr_grade_rank/LICENSE/LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+    OpenERP, Open Source Management Solution
+    Copyright (C) 2011-TODAY Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>).
```
Comment (ForamSerpentCS) [inline hr_grade_rank/LICENSE/LICENSE]: 2020-TODAY

File: hr_grade_rank/models/hr_grade_rank.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
```
Comment (ForamSerpentCS) [inline hr_grade_rank/models/hr_grade_rank.py]: Imports should be in alphabetical order.

File: hr_grade_rank/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "license": "AGPL-3",
+    "summary": "Manage grade and rank of employee",
+    "author": "Serpent Consulting Services Pvt. Ltd.",
+    "maintainer": "Serpent Consulting Services Pvt. Ltd.",
+    "category": "Human Resources",
+    "website": "http://www.serpentcs.com",
+    "depends": ["hr"],
+    "data": ["security/ir.model.access.csv", "views/hr_grade_rank_view.xml"],
+    "images": ["static/description/HRGradeRank.png"],
+    "licence": "AGPL-3",
```
Comment (ForamSerpentCS) [inline hr_grade_rank/__manifest__.py]: Exclude recurrence of attribute 'licence'.

File: hr_grade_rank/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "sequence": 1,
+    "installable": True,
+    "auto_install": False,
```
Comment (ForamSerpentCS) [inline hr_grade_rank/__manifest__.py]: We can remove 'auto_install' attribute as it has False value by default.

**Review by ForamSerpentCS — CHANGES_REQUESTED** [review]

**Comment by pragati-patel98** [comment]
@ForamSerpentCS I solved the comments, Can you please review it?

**Review by ForamSerpentCS — APPROVED** [review]

**Pull request merged**